library/src/main/scala/scala/scalajs/macroImpls/JSMembers.scala

```diff
@@ -0,0 +1,87 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js API               **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013, LAMP/EPFL        **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-lang.org/     **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+
+
+package scala.scalajs.macroImpls
+
+import Compat210._
+
+/** JSMember is an ADT more or less equivalent to Scala's MethodType
+ *  that allows to distinguish setters and getters from methods.
+ *
+ *  It also allows to check method conformance based on Scala.js' JavaScript
+ *  calling conventions.
+ *
+ *  Currently does not support polymorphic method types.
+ *
+ *  @author Tobias Schlatter
+ */
+trait JSMembers {
```

Comment:
Ah, indeed... I thought `UseAsMacros` extends this so it would need to be public... But it doesnt :)